src/main/java/mate/jdbc/service/impl/impl/DriverServiceImpl.java

```diff
@@ -0,0 +1,40 @@
+package mate.jdbc.service.impl.impl;
+
+import java.util.List;
+import mate.jdbc.dao.DriverDao;
+import mate.jdbc.dao.impl.DriverDaoIml;
+import mate.jdbc.lib.Inject;
+import mate.jdbc.lib.Service;
+import mate.jdbc.model.Driver;
+import mate.jdbc.service.impl.DriverService;
+
+@Service
+public class DriverServiceImpl implements DriverService {
+    @Inject
+    private DriverDao driverDao = new DriverDaoIml();
```

Comment:
the idea of dependency injections is to get rid of  using constructor with `new` in the classes.
```suggestion
    private DriverDao driverDao;
```